content/en/blog/the-time-is-now/index.md

```diff
@@ -0,0 +1,100 @@
+supports currently both.</em>
+{{< /imgproc >}}
+
+However, choosing a symmetric protocol may widen the spectrum of your use cases.
+Participants can execute different functions during the interaction in a symmetric protocol.
+So, for example, in the user authentication scenario, in addition to the service (server)
+authenticating the user (client) based on verified data, the individual could
+also get a verification from the service-operating organization of its authenticity.
+
+## Integrate Carefully
+
+Whichever stack you choose, there are two things to keep in mind
+
+1. Implement the integration to the library or service providing SSI functionality following
+well-known modular principles so that it is easy to replace with a different library or service.
+1. Do not let the underlying technology details pollute your application code.
+Thus, ensure the SSI tooling you use hides the implementation details related
+to credentials formats and protocol messages from your application.
+This approach ensures that changes, for example, to the credential data format,
+have a manageable effect on your application.
+
+<br><img src="https://github.com/findy-network/agency-demo/raw/master/client/public/seo-logo.jpg?raw=truef" width="800px" /><br>
+
+*Decentralized Identity Demo is a cool SSI simulation. Its original developer is [Animo](https://animo.id/),
+and the SSI functionality was done using [aries-framework-javascript](https://github.com/hyperledger/aries-framework-javascript).
+As an experiment for SSI technology portability, I took
+the open-sourced codes and converted the application to use [Findy Agency](https://findy-network.github.io/)
+instead of AFJ. You can find both [the original codes](https://github.com/animo/animo-demo)
+and [my version](https://github.com/findy-network/agency-demo) in GitHub.*
+
+The best option is to pick open-source tooling so that if some functionality is missing,
+you can roll up your sleeves and implement it yourself. If this is not an option,
+select a product that ensures compatibility with multiple flavors of SSI credentials
+and data exchange protocols so that interoperability with other parties is possible.
+After all, if there is no interoperability, there is no decentralization.
+
+## The Time Is Now
+
+The recent advancements in LLM technology and ChatGPT offer a glimpse of what's possible
+when large audiences can suddenly access new technology that provides them added value.
+SSI can potentially revolutionize digital services similarly, and organizations
+should prepare for this paradigm shift. Is your organization ready when this transformation takes place?
+
+In conclusion, now is the time to plan, experiment and build SSI capabilities. Begin
+by understanding the high-level building blocks and experimenting with the technology
+that best aligns with your use case. The technology will mature in time, and
+the application layer will be unaffected if implemented correctly.
```

Comment:
I was just marking it here. We only have an application layer if we have an overall layer model. Findy Agency at least tries to bring a layer model with it.